### ISSUE TITLE
404 error

### ISSUE THREAD

**Issue by Yashneel123** [InterstellarNetwork/Interstellar#486]
There is a 404 error it is not working when I open up the link by clicking in github codespaces
**To Reproduce**

1. Go to Github Codespace
2. Click on the port
3. It says 404 error

I expected the Interstellar to work but it is not working

**Screenshots**

My device is school chromebook and it says the site is insecure and not found

- Device: [Ex. Chromebook, iPad, etc.]
- OS [Ex. ChromeOS, Windows, iOS
- Browser [Ex. Chrome, Safari]

**Additional context**
Pls just fix it its not working now i cant play roblox with my friends
![Obsucure space orbit](https://github.com/InterstellarNetwork/Interstellar/assets/131425784/05e53026-c18d-4edb-92c0-e2c721c88b85)

**Comment by Yashneel123** [comment]
Pls help me it is only not working for  me

**Comment by bruhwhyamisobad** [comment]
Did you open the port?

**Comment by xbubbo** [comment]
Set the port properly and turn the visibility to public.

**Comment by Yashneel123** [comment]
Yeah I didn't open the port properly but I have another question.

**Comment by Yashneel123** [comment]
Um when ever I open a port it says webpage not found
